bazel/grpc_deps.bzl

```diff
@@ -115,9 +115,9 @@ def grpc_deps():
             url = "https://boringssl.googlesource.com/boringssl/+archive/afc30d43eef92979b05776ec0963c9cede5fb80f.tar.gz",
         )
 
-    if "com_github_madler_zlib" not in native.existing_rules():
```

Comment:
why exactly is this change needed? It looks like you're only renaming the library. Is that because we need to use the same name that protobuf uses?

Reply:
Yea. Because protobuf somehow expects @zlib//:zlib to be present, and I don't want to build zlib twice: https://github.com/protocolbuffers/protobuf/blob/master/BUILD#L22